lib/uo_cli.rb

```diff
@@ -184,11 +192,20 @@ def self.create_scenario_csv_file(feature_file_path, feature_id)
     # 
     # Folder gets created in the current working directory
     # Includes weather for UO's example location, a base workflow file, and mapper files to show a baseline and a high-efficiency option.
-    def self.create_project_folder(dir_name, empty_folder = false)
-        if Dir.exist?(dir_name)
-            abort("ERROR:  there is already a directory here named #{dir_name}... aborting")
-        else
-            puts "CREATING URBANopt project directory: #{dir_name}"
+    def self.create_project_folder(dir_name, empty_folder = false, overwrite_project = false)
+        if overwrite_project == true
+            if Dir.exist?(dir_name)
+                FileUtils.rm_rf(dir_name)
+                puts "Deleting existing contents of project directory: #{dir_name}"
+            else 
+                puts "Directory named #{dir_name} does not exist."
```

Comment:
I would remove this puts statement...it sounds like it's an error, but it's not.  the -o command just overwrite if the project folder exists, but it doesn't have to exist.

and I would change the first puts to "Overwriting existing project directory: #{dir_name}

Reply:
I just fixed this in commit below.